src/applications/simple-forms/shared/config/submit-transformer.js

```diff
@@ -1,9 +1,24 @@
 import { transformForSubmit as formsSystemTransformForSubmit } from 'platform/forms-system/src/js/helpers';
 
+const escapedCharacterReplacer = (_key, value) => {
+  if (typeof value === 'string') {
+    return value
+      .replaceAll('"', "'")
+      .replace(/(?:\r\n|\n\n|\r|\n)/g, '; ')
+      .replace(/(?:\t|\f|\b)/g, '')
+      .replace(/\\(?!(f|n|r|t|[u,U][\d,a-fA-F]{4}))/gm, '/');
```

Comment:
do you think we should keep the old logic for this line and just replace ANY backslash? vs only backslashes followed by a certain set of characters? i imagine backslash could also affect other things also unintentionally.

Reply:
Ohhh yeah, I remember the tricky part. The last part of that replace line matches against u or U + 4 unicode characters. I wasn't sure what we should do if we're given those since we proved (sort of?) that escaped unicode characters work. 

@Thrillberg Do you think we should let unicode characters through or should we just drop them? I know it "worked" when we tested in the engineering sync last week, but did it work out of the box or did it only work when we escaped it like 5 times?